back-end/server/api/activity.js

```diff
@@ -31,7 +31,21 @@ router.get("/weekly/:userId", async (req, res, next) => {
   } catch (err) {
     next(err)
   }
+})
 
+router.get('/:activityId/ad', async (req, res, next) => {
+  try {
+    const adData = await Ad.find({
+      where: {
+        activityId: +req.params.activityId,
+      },
+    })
+    console.log(adData)
+    const ad = adData.dataValues
```

Comment:
You don't need to store it in a variable by calling dataValues on it. You can just do res.json(adData)